clang/runtime/dpct-rt/include/sparse_utils.hpp

```diff
@@ -294,6 +294,81 @@ class optimize_info {
 #endif
 
 #ifdef __INTEL_MKL__ // The oneMKL Interfaces Project does not support this.
+namespace detail {
+#ifdef DPCT_USM_LEVEL_NONE
+#define SPARSE_CALL(X, Y) X;
+#else
+#define SPARSE_CALL(X, Y)                                                      \
+  sycl::event e = X;                                                           \
```

Comment:
Line 302 can be removed by inline X in the call.

Reply:
done